src/Features/CSharp/Organizing/Organizers/MemberDeclarationsOrganizer.Comparer.cs

```diff
@@ -171,6 +171,21 @@ private OuterOrdering GetOuterOrdering(MemberDeclarationSyntax x)
                         return OuterOrdering.Remaining;
                 }
             }
+
+            private static bool ShouldCompareByName(MemberDeclarationSyntax x)
+            {
+                // Constructors, destructors, indexers and operators should not be sorted by name.
+                switch (x.Kind())
+                {
+                    case SyntaxKind.ConstructorDeclaration:
+                    case SyntaxKind.DestructorDeclaration:
+                    case SyntaxKind.IndexerDeclaration:
+                    case SyntaxKind.OperatorDeclaration:
```

Comment:
I could add it explicitly here, but it's not necessary, because GetNameToken will return default(SyntaxToken) for it.